### PR TITLE
Fixed MonoGame.Extended.Conten.Pipeline compilation on Linux

### PR DESCRIPTION
**Pull request by cra0zy**

### CODE REVIEW

File: Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj [hunk 1]
```diff
@@ -97,7 +97,7 @@
     <Compile Include="Tiled\TmxTerrain.cs" />
     <Compile Include="Tiled\TmxTile.cs" />
     <Compile Include="Tiled\TmxTileOffset.cs" />
-    <Compile Include="Tiled\TmxTileset.cs" />
+    <Compile Include="Tiled\TmxTileSet.cs" />
```
Comment (craftworkgames) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: I think [tileset is one word](https://www.google.com.au/search?q=tileset) so it should have a lowercase s. Was this stopping the project building on Linux?

Comment (craftworkgames) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: Ah, I see the problem. The [filename doesn't match the class name](https://github.com/craftworkgames/MonoGame.Extended/blob/master/Source/MonoGame.Extended.Content.Pipeline/Tiled/TmxTileSet.cs).

Comment (cra0zy) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: Want me to change the filename instead?

Comment (craftworkgames) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: It's okay I've already done it. I don't have a Linux machine to test on though please make sure it works.

Comment (cra0zy) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: You did something silly....

You merged this and changed the filename so now you get the exact same problem, but now you reversed where the big letter occurs....

Comment (craftworkgames) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: I was pretty sure I checked that they where the same. I think there's something about the way Windows handles it or maybe the windows version of git. Even though I had renamed the file the change didn't show up in the diff. Anyway, you know what to do. I'll leave it in your capable hands.

Comment (cra0zy) [inline Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj]: Aye Aye Captain.

NTFS and FAT filesystems aren't case sensitive.

**Comment by cra0zy** [comment]
@craftworkgames Do not modify a single thing in samples, I'm gonna do a PR to make it crossplatform.

**Comment by craftworkgames** [comment]
> Do not modify a single thing in samples, I'm gonna do a PR to make it crossplatform.

@cra0zy Okay no problem.

Just be aware that I've always had the cross-platform issue in the back of my mind.  The WinForms fluff around the outside of the samples was only ever intended to be a temporary thing until we get some sort of GUI in the library. Then I'd like to refactor it to use a GUI system built on MonoGame. 

That said, if it's not a lot of work and you're keen to do it by all means go ahead. What tech are you going to use?

**Comment by cra0zy** [comment]
Xwt.... I am actually mostly done, just to check if everything works on WIndows.

**Comment by cra0zy** [comment]
You know, now that I think about it using MonoGame main window will be better, I'll do it tomorrow morning.

**Comment by craftworkgames** [comment]
You might as well submit the PR if it's ready to go.

**Pull request merged**